test/e2e/pages/update-move.js

```diff
@@ -0,0 +1,53 @@
+import { Selector, t } from 'testcafe'
+
+import CreateMovePage from './create-move'
+import MoveDetailPage from './move-detail'
+
+const moveDetailPage = new MoveDetailPage()
+
+class UpdateMovePage extends CreateMovePage {
+  constructor(id) {
+    super()
+    this.url = `/move/${id}/edit`
+    this.fields = {
+      ...this.fields,
+      policeNationalComputerReadOnly: Selector(
+        '[type=hidden][name=police_national_computer]'
+      ),
+      policeNationalComputerHeading: Selector('.app-read-only-field__heading'),
```

Comment:
Maybe these should live under a different property? They're not technically a "field"

Reply:
Do you have a suggestion?

Reply:
So there was a pattern of `this.nodes` that had been used elsewhere. Maybe use that for now. But they definitely don't fit into `fields`...

Reply:
Move non-fields to `this.nodes`